cluster-provision/k8s/1.17/manifests/cnao/operator.yaml

```diff
@@ -153,13 +153,13 @@ spec:
         - name: KUBEMACPOOL_IMAGE
           value: quay.io/kubevirt/kubemacpool:v0.8.3
         - name: MACVTAP_CNI_IMAGE
-          value: quay.io/kubevirt/macvtap-cni:v0.1.0
+          value: quay.io/kubevirt/macvtap-cni:v0.2.0
```

Comment:
why do we have to bump individual components as well ? we are bumping cnao, which already has pinned versions of the upstream components. This feels redundant.

Expanding on that; what if there's a mismatch between the number I set here and the version pinned in CNAO ?

@phoracek @qinqon ^

Reply:
> why do we have to bump individual components as well ? we are bumping cnao, which already has pinned versions of the upstream components. This feels redundant.

The operator definition is the one carrying the image names, so they can be easily overwritten. e.g. on D/S.

> 
> Expanding on that; what if there's a mismatch between the number I set here and the version pinned in CNAO ?

Since manifests are shipped in CNAO image, it can happen that you use manifests meant for component version X with component image version Y.

Reply:
Thanks.